svgnative/src/SVGDocumentImpl.cpp

```diff
@@ -835,6 +860,10 @@ void SVGDocumentImpl::ParseGraphicsProperties(GraphicStyleImpl& graphicStyle, co
 
 float SVGDocumentImpl::ParseColorStop(XMLNode* node, std::vector<ColorStopImpl>& colorStops, float lastOffset)
 {
+    /* XXX */
```

Comment:
What is this comment?

Reply:
For the void functions, my patch returns immediately when invalid node is given.
But, ParseColorStop() should returns the floating number, so my patch returns 0, it would be hard for the client to find that: the node is broken and returned "0" is the fallbacked value.
OK, I would replace this by assertion.